examples/complete/outputs.tf

```diff
@@ -0,0 +1,8 @@
+output "groups" {
```

Comment:
Do we need this file and/or outputs? Prob the outputs should only be anything needed by the tests(which is nothing rn I think).